circuits/ts/index.ts

```diff
@@ -23,13 +24,20 @@ const cleanThreads = async () => {
     return promises
 }
 
-const genProof = (
+const genProof = async (
     inputs: string[],
-    rapidsnarkExePath: string,
-    witnessExePath: string,
     zkeyPath: string,
+    rapidsnarkExePath?: string,
+    witnessExePath?: string,
+    wasmPath?: string,
     silent = true,
-): any => {
+): Promise<any> => {
```

Comment:
Why do we return `any` here? Can we not be sure of the interface of the return values?

Reply:
yes I think we can sort all that out when doing the actual refactor